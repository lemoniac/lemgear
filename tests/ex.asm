	ld A, 5
	ex AF, AF'
	cp 0
	jr z, l
	halt
l:
	ld A, 10
	ld B, 3
	ex AF, AF'
	
	ld DE, $1234
	ld HL, $5678
	ex DE, HL

	halt
	
	
	

