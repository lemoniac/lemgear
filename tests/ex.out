AF : 0500
BC : 0300
DE : 5678
HL : 1234
